Enhancement to default to PV instead of HVM on Xen host.
Index: virt-manager-2.2.0/virtManager/createvm.py
===================================================================
--- virt-manager-2.2.0.orig/virtManager/createvm.py
+++ virt-manager-2.2.0/virtManager/createvm.py
@@ -740,7 +740,12 @@ class vmmCreateVM(vmmGObjectUI):
         if gtype is None:
             # If none specified, prefer HVM so install options aren't limited
             # with a default PV choice.
+            # If xen connection, favor PV installation
+            conn_is_xen = self.conn.is_xen()
             for g in self.conn.caps.guests:
+                if conn_is_xen and g.os_type == "xen":
+                    gtype = "xen"
+                    break
                 if g.os_type == "hvm":
                     gtype = "hvm"
                     break
